packages/os/bootstrap-containers@.service

```diff
@@ -7,10 +7,17 @@ Wants=host-containers.service
 # started by systemd
 RefuseManualStart=true
 RefuseManualStop=true
+# If a sentinel file exists for this bootstrap container, it means we should skip
+# since we've ran this bootstrap container already.
+ConditionPathExists=!/run/bootstrap-containers/%i.ran
 
 [Service]
 Type=oneshot
 EnvironmentFile=/etc/bootstrap-containers/%i.env
+# Create a sentinel file to mark that we've ran
+ExecStart=/usr/bin/mkdir -p /run/bootstrap-containers/
```

Comment:
nit: I have a slight preference to do this `mkdir` via `bootstrap-containers-tmpfiles.conf` so we don't repeat the work for N bootstrap containers.